workflow-templates/maven.properties.json

```
@@ -0,0 +1,8 @@
+{
+    "name": "Build using Maven",
+    "description": "Build using Maven.",
+    "iconName": "maven",
+    "categories": [
+        "Maven POM"
+    ]
```

I think we can assume any target repo will have a top-level POM

```suggestion
    ],
    "filePatterns": [
        "pom.xml$"
    ]
```